unstructured/ingest/connector/sql/schema.py

```diff
@@ -0,0 +1,90 @@
+import json
```

Comment:
Where is this file used? I'd like to avoid having any code around creating a schema in the core ingest directory. This can live in either the test or the docs directory perhaps.